blocks/highlight/highlight.css

```diff
@@ -10,7 +10,7 @@
   color: var(--color-text);
 }
 
-.block.highlight > * {
+.block.highlight:not(.featured) > * {
```

Comment:
How about 
```
.block.highlight.featured > * {
  //whatever
}
```

Reply:
<img width="1630" alt="image" src="https://github.com/hlxsites/sunstar-foundation/assets/37831850/abedcd0f-ff53-4eca-b9b2-cf5b6d551398">

Sure. How about leveraging the below then - 

.block.highlight > * {
  flex: 1;
}

.block.highlight.featured > div:first-child {
  flex: 1;
}

.block.highlight.featured > div:last-child {
  flex: 0;
}

Reply:
ok